orderer/common/multichannel/metrics.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package multichannel
+
+import (
+	"github.com/hyperledger/fabric/common/metrics"
+	"github.com/hyperledger/fabric/orderer/common/types"
+)
+
+var (
+	clusterStatusOpts = metrics.GaugeOpts{
+		Namespace:    "cluster",
+		Subsystem:    "",
+		Name:         "status",
+		Help:         "The status of the cluster node: 0 if inactive, 1 if active, 2 if onboarding.",
```

Comment:
I'm not totally convinced here, but wouldn't it be more natural to go 0 inactive, 1 onboarding, 2 active? (ie, typically you transition from 0 to 1 to 2?)

Reply:
That's how I originally implemented this but decided against it due to the fact that we may end up adding different options in the future. For the sake of keeping a stable base set of statuses and allowing new ones in the future, I thought it might be best to move away from the concept of a progression in the enum.